mesonbuild/compilers.py

```diff
@@ -2729,3 +2733,135 @@ def unix_compile_flags_to_native(self, args):
 
     def get_link_debugfile_args(self, targetfile):
         return []
+
+
+# Only tested on linux for ICC 17.0.0
+#    * object extraction does not compile (don't know) (25 object extraction fails)
```

Comment:
It is not us. ICC is writing the depfile itself and it quotes spaces incorrectly. This needs to be fixed in ICC, we can't do anything about it.